kymatio/scattering2d/backend/tensorflow_backend.py

```diff
@@ -32,7 +31,7 @@ def __call__(self, x):
         else:
             paddings = [[0, 0]] * len(x.shape[:-2])
             paddings += [[self.pad_size[0], self.pad_size[1]], [self.pad_size[2], self.pad_size[3]]]
-            return tf.cast(tf.pad(x, paddings, mode="REFLECT"), tf.complex64)
+            return tf.pad(x, paddings, mode="REFLECT")
```

Comment:
the cast removed above is not necesary?

Reply:
Yup! The whole reason for the refactor was to move this casting to complex business outta the padding functions!

Reply:
I suppose you could also say that its been refactored in the rfft call below